pycloudflare/utils.py

```diff
@@ -0,0 +1,32 @@
+from functools import wraps
+
+from pycloudflare.exceptions import SSLUnavailable
+from pycloudflare.services import HTTPServiceError
+
+
+errors_mapping = {
+    1001: SSLUnavailable,
+}
```

Comment:
> I was assuming the ID and exception would be parameters to the wrapper.

I tried this approach and didn't like it, because it doesn't allow to translate several possible exceptions.